src/crate/theme/rtd/crate/static/vendor/furo/scripts/furo.js

```diff
@@ -71,14 +71,12 @@ function scrollHandler(positionY) {
 // Theme Toggle
 ////////////////////////////////////////////////////////////////////////////////
 function setTheme(mode) {
-  if (mode !== "light" && mode !== "dark" && mode !== "auto") {
-    console.error(`Got invalid theme mode: ${mode}. Resetting to auto.`);
-    mode = "auto";
+  if (mode !== "light" && mode !== "dark") {
+    mode = "light";
   }
 
   document.body.dataset.theme = mode;
   localStorage.setItem("theme", mode);
-  console.log(`Changed to ${mode} mode.`);
```

Comment:
Do we want to make changes to the vendored assets? Are those updates coming from upstream instead?

Reply:
A few of them I had to change, because some of the logic comes from config files which aren't in our package yet. We only have light and dark mode (where auto is basically registered on the first visit and set to the systems default) instead of light/dark/auto.

Reply:
Ok. Thanks for clarifying.